index.html

```diff
@@ -113,19 +113,19 @@ <h1 id="hackdavis">HACK<b>DAVIS</b></h1>
     <section class="stats-container">
       <div class="container stats-content">
           <ul class="list-features justify-content-center row">
-            <li class="col-lg-3 col-sm-6" data-entrance="fade">
```

Comment:
We should not have two column layout on mobile. Two column layout between mobile and tablet was a concession to reduce the size in pixels that the stats was taking up and provide a smoother transition between tablet and mobile.

I am okay with having the user scroll through the stats. On tablet, text was large enough to draw attention with two columns. Two columns on mobile makes the text small, plus you see the lines more than you see the text. Having a small 4 panel box also distracts you from reading the Hack for Social Good header and text, which will take up a significantly larger amount of the screen